native/src/components/PrivacyCheckbox.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import styled from 'styled-components/native'
+
+import buildConfig from '../constants/buildConfig'
+import openExternalUrl from '../utils/openExternalUrl'
+import Link from './Link'
+import { SnackbarType } from './SnackbarContainer'
+import Checkbox from './base/Checkbox'
+
+const FlexContainer = styled.Pressable`
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  flex: 1;
+`
+
+const StyledLink = styled(Link)`
+  color: ${props => props.theme.colors.linkColor};
+  text-decoration: underline solid ${props => props.theme.colors.linkColor};
+  align-self: center;
```

Comment:
:wrench: I think this styling should be removed, we should just use the default `Link` styling (which is grey instead of blue).